recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -21,6 +21,11 @@ if "%CONDA_BLD_PATH%" == "" (
     set "CONDA_BLD_PATH=C:\\bld\\"
 )
 
+:: Increase pagesize, cf. https://github.com/conda-forge/conda-forge-ci-setup-feedstock/issues/155
+if "%CONDA_BLD_PATH%" == "C:\\bld\\" (
+    call recipe\SetPageFileSize.ps1 -MinimumSize 4GB -MaximumSize 8GB -DiskRoot "C:"
```

Comment:
Why C? C is where the building happens, so we want this to be on D

Reply:
Wasn't sure if `D:`'s always available. Thanks for catching this.